### PR TITLE
1.0.14_crab version of Use system environment for gfal2 stageout

### PR DESCRIPTION
**Pull request by emaszs**
After the merge we'd also like a new tag for our branch, 1.0.14_crab_8. Thanks.

### CODE REVIEW

File: src/python/WMCore/Storage/Backends/GFAL2Impl.py [hunk 1]
```diff
@@ -21,12 +21,12 @@ class GFAL2Impl(StageOutImpl):
...
         StageOutImpl.__init__(self, stagein)
         # If we want to execute commands in clean shell, we can`t separate them with ';'.
-        # Next commands after separation are executed without env -i and this leads us with
+        # Next commands after separation are executed without env -i and this leads us with 
         # mixed environment with COMP and system python.
         # GFAL2 is not build under COMP environment and it had failures with mixed environment.
-        self.setups = "env -i X509_USER_PROXY=$X509_USER_PROXY bash -c '%s'"
-        self.removeCommand = self.setups % '. $PWD/startup_environment.sh; printenv; date; gfal-rm -vvv -t 600 %s '
-        self.copyCommand = self.setups % '. $PWD/startup_environment.sh; printenv; date; gfal-copy -vvv -t 2400 -T 2400 -p %(checksum)s %(options)s %(source)s %(destination)s'
+        self.setups = "env -i X509_USER_PROXY=$X509_USER_PROXY sh -c '%s'"
```
Comment (juztas) [inline src/python/WMCore/Storage/Backends/GFAL2Impl.py]: why you change to sh?

**Comment by amaltaro** [comment]
CRAB version of #6808 (well, kind of...)
@juztas can you guys discuss and converge on a single fix please (e.g., why "sh" instead of "bash"? The master branch version uses dirty environment variables and sub-scripts .. this one looks neat.

**Comment by juztas** [comment]
relax Alan :D Emilis is working on it.

**Comment by amaltaro** [comment]
I'm never relaxed, LoL 
Ok, you guys let us know when it's ready to be merged (and in agree with the master :D)

**Comment by emaszs** [comment]
Ok, this PR is now even with the resulting files after #6808 is merged, sorry for the mess.

**Comment by amaltaro** [comment]
1.0.14_crab_8 tagged

Guys, I've a suggestion of tag naming. For WMAgent, we use e.g. 1.0.14.patch8. How about we start moving the crab tags closer to what we already do for WMAgent (just to avoid mistakes), so something like 1.0.14.crab8 in the future.

Can you discuss between your team to see whether there is any problem in adopting such convention for the next cycle?

**Pull request merged**